src/flamegraph.js

```diff
@@ -437,12 +437,14 @@ export default function () {
             g.exit()
                 .remove()
 
-            g.on('mouseover', function (d) {
-                if (tooltip) tooltip.show(d, this)
-                detailsHandler(labelHandler(d))
-            }).on('mouseout', function () {
+            g.on('mouseout', function () {
                 if (tooltip) tooltip.hide()
                 detailsHandler(null)
+            }).on('mousemove', function (d) {
+                const chartBoundingClientRect = svg._groups[0][0].parentNode.getBoundingClientRect()
```

Comment:
What `rect` is this exactly?

Reply:
Using the DOM API [getBoundingClientRect()](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect) to get the size and position of the chart area rendered in the web page.